articles/connections/social/dropbox.md

```diff
@@ -0,0 +1,81 @@
+---
+connection: Dropbox
+image: /media/connections/dropbox.png
+seo_alias: dropbox
+description: This page shows you how to connect your Auth0 app to Dropbox. You will need to generate keys, copy these into your Auth0 settings, and enable the connection.
+---
+
+# Connect your app to Dropbox
+
+To connect your Auth0 app to Dropbox, you will need to generate a *Client ID* and *Client Secret* in a Dropbox app, copy these keys into your Auth0 settings, and enable the connection.
+
+## 1. Login to the developer portal
+
+Go to the [Dropbox Developer](https://www.dropbox.com/developers) portal and log in with your Dropbox credentials. 
+
+Click **Create your app**:
+
+![](/media/articles/connections/social/dropbox/dropbox-01.png)
+
+## 2. Create your app
+
+1. Under **Choose an API**, select **Dropbox API**.
+2. Under **Choose the type of access you need**, select **App folder**.
```

Comment:
@Catografix I believe it could actually be either depending on what the developer is trying to do. You should try them both out to make sure they work correctly.